app/uk/gov/hmrc/brm/controllers/BirthEventsController.scala

```diff
@@ -16,100 +16,100 @@
 
 package uk.gov.hmrc.brm.controllers
 
+import javax.inject.Inject
 import org.joda.time.DateTime
-import play.api.data.validation.ValidationError
 import play.api.libs.json._
-import play.api.mvc.{Action, Request, Result}
+import play.api.mvc.{Action, ControllerComponents, Request, Result}
 import uk.gov.hmrc.brm.audit._
 import uk.gov.hmrc.brm.config.BrmConfig
 import uk.gov.hmrc.brm.filters.{Filter, Filters}
-import uk.gov.hmrc.brm.implicits.Implicits.{AuditFactory, MetricsFactory}
-import uk.gov.hmrc.brm.metrics.BRMMetrics
+import uk.gov.hmrc.brm.implicits.{AuditFactory, MetricsFactory}
+import uk.gov.hmrc.brm.metrics._
 import uk.gov.hmrc.brm.models.brm._
 import uk.gov.hmrc.brm.models.matching.MatchingResult
 import uk.gov.hmrc.brm.services.LookupService
-import uk.gov.hmrc.brm.utils.BRMLogger._
-import uk.gov.hmrc.brm.utils.{BirthResponseBuilder, HeaderValidator, _}
-import uk.gov.hmrc.brm.utils.CommonUtil._
-import scala.concurrent.Future
+import uk.gov.hmrc.brm.utils.{BRMLogger, BirthResponseBuilder, CommonUtil, HeaderValidator}
 import uk.gov.hmrc.http.HeaderCarrier
 
-object BirthEventsController extends BirthEventsController {
-  override val service = LookupService
-  override val countryAuditor = new WhereBirthRegisteredAudit()
-  override val auditFactory = new AuditFactory()
-  override val transactionAuditor = new TransactionAuditor()
-  override val matchingAuditor = new MatchingAudit()
-  override val headerValidator = HeaderValidator
-}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
 
-trait BirthEventsController extends BRMBaseController {
+class BirthEventsController @Inject()(val service: LookupService,
+                                      countryAuditor: WhereBirthRegisteredAudit,
+                                      auditFactory: AuditFactory,
+                                      config: BrmConfig,
+                                      val transactionAuditor: TransactionAuditor,
+                                      val matchingAuditor: MatchingAudit,
+                                      val headerValidator: HeaderValidator,
+                                      cc: ControllerComponents,
+                                      val commonUtils: CommonUtil,
+                                      val logger: BRMLogger,
+                                      val metrics: MetricsFactory,
+                                      filters: Filters,
+                                      implicit val engAndWalesMetrics: EnglandAndWalesBirthRegisteredCountMetrics,
+                                      implicit val northIreMetrics: NorthernIrelandBirthRegisteredCountMetrics,
+                                      implicit val scotlandMetrics: ScotlandBirthRegisteredCountMetrics,
+                                      implicit val invalidRegMetrics: InvalidBirthRegisteredCountMetrics) extends BRMBaseController(cc) {
```

Comment:
I understand why this is here but I still want to say, this is an impressive amount of injections. it does call for a separate clean-up ticket